frontend/src/utils/api.js

```diff
@@ -1,8 +1,9 @@
 // utility functions //
 window.$ = (id) => document.getElementById(id);
 window.enquery = (key, val) => `${key}=${encodeURIComponent(val)}`;
-window.BASE_URL = 'https://github.com';
+window.REPOSENSE_REPO_URL = 'https://github.com/reposense/RepoSense';
 window.HOME_PAGE_URL = 'https://reposense.org';
+window.UNSUPPORTED_INDICATOR = 'UNSUPPORTED';
```

Comment:
The links are only generated in the frontend so that each repository only needs reponame, domain and org, together with the URL templates. This constant was just a way for the backend to signal to the frontend that this specific link is not supported, in which case it returns an `undefined`. The `undefined` is then set to the `href` which causes the link to be disabled. I'm not sure if there's a better way to go about this though suggestions are greatly appreciated.

Reply:
I see, I think I got confused about this part. Seems like the code you have already achieves the desired result.